### ISSUE TITLE
Error: Unkown value 3,3,3,3 for padding Enum in  tf.loadModel

### ISSUE THREAD

**Issue by scaelles** [tensorflow/tfjs#84]
Hello,

I have exported successfully my Keras model using the tensorflowjs_converter without any error or warning. However, when I try to load the model using tf.loadModel('model.json'), I get the following error:
```
Uncaught (in promise) Error: Unkown value 3,3,3,3 for padding Enum
    at new t (tf.min.js:1)
    at i (tf.min.js:1)
    at i (tf.min.js:1)
    at i (tf.min.js:1)
    at i (tf.min.js:1)
    at Object.i [as convertPythonicToTs] (tf.min.js:1)
    at tf.min.js:1
    at r (tf.min.js:1)
    at Object.next (tf.min.js:1)
    at tf.min.js:1

```
I use the following Keras layers: BatchNormalization, Conv2D, Activation('relu'), ZeroPadding2D, MaxPooling2D. Also the following Keras operation: keras.layers.add. Is this due to a layer not implemented yet in tensorflowjs?
I am using tensorflow 1.7, Keras 2.1.5 and tensorflowjs 0.1.0
Also, I found [here](https://github.com/tensorflow/tfjs-converter/blob/master/docs/supported_ops.md) the operations and layers supported from Tensorflow, where could I find the same for Keras?

Thanks a lot

**Comment by nsthorat** [comment]
Hi @scaelles, thanks for the bug.

Could you upload your model.json (github gist) for us to take a look at?

**Comment by scaelles** [comment]
Thanks a lot for your fast reply @nsthorat!
Here is the gist with the [model.json](https://gist.github.com/scaelles/6917c82ad30d40609a8dc98d753c0881). Since I posted the issue, I have found out that if I remove the ZeroPadding2D layers of the Keras model, the error is no longer there, so I guess it has something to do with the conversion of this layer.

**Comment by nsthorat** [comment]
Ah, thanks! Adding another person who may be able to get a fix in (who wrote much of the serialization logic).

**Comment by ericdnielsen** [comment]
I've identified two issues:
1) ZeroPadding2D wasn't ported.
2) The error message is bad, because all layers that were ported use the padding keyword to map to a string (same, valid, casual) so we converted padding to an Enum internally for stronger type.  The ZeroPadding2D layers uses 'padding' as a key to a multi-dimensional array, and thus breaks the mapping of value to enum.

**Comment by nsthorat** [comment]
This fix will go out with 0.7.1.

**Comment by scaelles** [comment]
Perfect, thanks a lot for fixing it so quickly!
